protocols/v2/binary-sv2/no-serde-sv2/codec/src/codec/mod.rs

```diff
@@ -1,9 +1,59 @@
-// At lower level I generally prefer to work with slices as more efficient than Read/Write streams
-// eg: Read for & [u8] is implemented with memcpy but here is more than enough to just get a
-// pointer to the original data
-
-// ANche se enum decode sarebbe faclie da implementare non viene fatto dato che ogni messaggio puo
-// essere derivato dal suo numero!
+/// Provides traits and utilities for encoding and decoding data at a low level,
```

Comment:
where is this header being rendered?

I cannot find it anywhere when I do `cargo doc`, I think it's pointless to have this with `///`